activejob/lib/active_job/exceptions.rb

```diff
@@ -145,6 +148,11 @@ def determine_delay(seconds_or_duration_or_algorithm:, executions:, jitter: nil)
         end
       end
 
+      def determine_jitter_for_delay(delay, jitter)
+        return 0.0 if jitter == 0
```

Comment:
```suggestion
        return 0.0 if jitter.zero?
```

Reply:
👍 Fixed in https://github.com/rails/rails/pull/38003/commits/fff3cd7b890fcc5f8502062201212f3cee911fb2. Thanks.